qiskit/compiler/transpile.py

```diff
@@ -631,14 +631,17 @@ def _parse_scheduling_method(scheduling_method, num_circuits):
     return scheduling_method
 
 
-def _parse_instruction_durations(backend, inst_durations, dt, scheduling_method, num_circuits):
+def _parse_instruction_durations(backend, inst_durations, dt, num_circuits):
     durations = None
-    if scheduling_method is not None:
-        from qiskit.transpiler.instruction_durations import InstructionDurations
-        if backend:
-            durations = InstructionDurations.from_backend(backend).update(inst_durations, dt)
-        else:
-            durations = InstructionDurations(inst_durations, dt)
+    if inst_durations is None and backend:
```

Comment:
Now `InstructionDurations` is always created if backend is specified even when it is not used (`scheduling_method=None`). Is it too tiny overhead to care?

Reply:
yeah i think it's tiny